src/cli.ts

```diff
@@ -61,6 +66,19 @@ if (configResults) {
   Object.assign(options, configResults.config)
 }
 
+if (cli.flags.exportConfig) {
+  if (options.typescript) {
+    const config = getTSLintConfig(options)
+    fs.writeFileSync('./tslint.json', JSON.stringify(config, null, 2))
+    console.log('tslint.json generated')
+  } else {
+    const config = getESLintConfig(options)
```

Comment:
You'll also want to get the `ignores` now by calling `getESLintIgnores(options.ignore)`.

This function returns an array of strings like:

```ts
['node_modules', 'dist/**', '**/fixtures', ...]
```

So in addition to creating the `eslintrc.json` file, you'll also want to create a `.eslintignore` file with these ignores. I believe it follows the same format as `.gitignore`, so the desired output would be:

**.eslintignore**

```
node_modules
dist/**
**/fixtures
```